modules/custom/openy_search/openy_google_search/openy_google_search.install

```diff
@@ -4,3 +4,106 @@
  * @file
  * Module installation file.
  */
+
+use Drupal\node\NodeInterface;
+use Drupal\pathauto\PathautoState;
+
+/**
+ * Implements hook_install().
+ */
+function openy_google_search_install() {
+
+  $alias = '/search';
+  /** @var \Drupal\Core\Path\AliasStorageInterface $path_alias_storage */
+  $path_alias_storage = \Drupal::service('path.alias_storage');
+  if ($path_alias_storage->aliasExists($alias, 'en')) {
+    return;
+  }
+  /** @var \Drupal\Core\Extension\ModuleInstallerInterface $module_installer */
+  $module_installer = \Drupal::service('module_installer');
```

Comment:
Why didn't you define a similar variable for `\Drupal::database()` which is used several times?